db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_vdc_networks.rb

```diff
@@ -0,0 +1,47 @@
+class AvailableVdcNetworks
+  def initialize(handle = $evm)
+    @handle = handle
+  end
+
+  def main
+    fill_fialog_field(fetch_list_data)
+  end
+
+  def fetch_list_data
+    service = @handle.root.attributes["service_template"] || @handle.root.attributes["service"]
+    vdc_networks = service.try(:orchestration_manager).try(:cloud_networks).try(:select) do |net|
+      net.type == "ManageIQ::Providers::Vmware::NetworkManager::CloudNetwork::OrgVdcNet"
```

Comment:
`net.kind_of? ManageIQ::Providers::Vmware::NetworkManager::CloudNetwork::OrgVdcNet` is preferred since storing the class name in the type column is an implementation detail of STI.